data-es6.js

```diff
@@ -4275,13 +4419,31 @@ exports.tests = [
         iojs:        true,
       },
     },
-    'WeakMap[Symbol.species]': {
+    'WeakMap.prototype.clear is missing': {
       exec: function () {/*
-        var prop = Object.getOwnPropertyDescriptor(WeakMap, Symbol.species);
-        return 'get' in prop && WeakMap[Symbol.species] === WeakMap;
+        return !("clear" in WeakMap.prototype);
```

Comment:
I feel like this test should only return "false" if both `WeakMap` is present, and `WeakMap#clear` is present - in any other case, this security hole doesn't exist.